src/accounts/accounts.utils.ts

```diff
@@ -46,7 +46,6 @@ export async function getBoostFile(chain: Chain): Promise<BoostData | null> {
     const boostFile = await getObject(REWARD_DATA, `badger-boosts-${parseInt(chain.chainId, 16)}.json`);
     return JSON.parse(boostFile.toString('utf-8'));
   } catch (err) {
-    console.warn({ message: `${chain.name} missing boost file`, err });
```

Comment:
removed since this can be a thing and until we have a detection for chains that need it this warning is just loud in logs